examples/rp/src/bin/gpout.rs

```diff
@@ -0,0 +1,34 @@
+#![no_std]
+#![no_main]
+#![feature(type_alias_impl_trait)]
+
+use defmt::*;
+use embassy_executor::Spawner;
+use embassy_rp::{clocks, pac};
```

Comment:
`embassy_rp::pac` is only available behind the `unstable-pac` Cargo feature. `embassy_rp::clocks` should reexport all the needed enums, so the user can do everything without `embassy_rp::pac`.

Reply:
Done, but since naming is the hardest thing in our profession I'd appreciate a sanity check on what I picked.